arch/arm/core/swi_tables.ld

```diff
@@ -3,6 +3,6 @@
  *
  * SPDX-License-Identifier: Apache-2.0
  */
-#if LINKER_ZEPHYR_FINAL
+#if LINKER_ZEPHYR_FINAL && CONFIG_ISR_TABLES_LOCAL_DECLARATION
 INCLUDE zephyr/isr_tables_swi.ld
```

Comment:
Fix should be to remove the zephyr prefix since this file will resize in the zephyr folder and would be a double prefix. Ninja (wrongly) adds a workaround for this, make does not, let's make the proper fix

Reply:
and it still means the LTO compatible ISR feature introduced in #66392 is not working with Makefiles.

We should not have some features only working with a specific generator.

Reply:
The zephyr prefix removed.